reports.jenkins.io.tf

```
@@ -0,0 +1,29 @@
+## This file contains the resources associated to the buckets used to store private and public reports
+resource "azurerm_resource_group" "prod_reports" {
+  name     = "prod-reports"
+  location = "East US 2"
```

```suggestion
  location = var.location
```
https://github.com/jenkins-infra/azure/blob/main/variables.tf#L3